src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -53,6 +56,7 @@ public class FailureDetector implements IFailureDetector, FailureDetectorMBean
     private static final Logger logger = LoggerFactory.getLogger(FailureDetector.class);
     public static final String MBEAN_NAME = "org.apache.cassandra.net:type=FailureDetector";
     private static final int SAMPLE_SIZE = 1000;
+    private static final long DEFAULT_FD_INITIAL_VALUE_MS = Gossiper.intervalInMillis * 2;
```

Comment:
In my understanding, `FailureDetector` and `Gossiper` are initialized almost at the same time because they have the following JMX bean names:
`public static final String MBEAN_NAME = `"org.apache.cassandra.net:type=Gossiper";`
`public static final String MBEAN_NAME = "org.apache.cassandra.net:type=FailureDetector";`

But I think you're right in general. These types of pull requests shouldn't raise any difficult questions when they got reviewed, so let's leave it for other PRs.

I've removed this constant.